include/nuttx/video/fb.h

```diff
@@ -569,6 +583,24 @@ struct fb_vtable_s
                FAR const struct fb_overlayblend_s *blend);
 # endif
 #endif
+
+  /* Specific Controls ******************************************************/
+
+  /* Set the frequency of the framebuffer update panel (0: disable refresh) */
+
+  int (*setframerate)(FAR struct fb_vtable_s *vtable, int rate);
+
+  /* Get the frequency of the framebuffer update panel (0: disable refresh) */
+
+  int (*getframerate)(FAR struct fb_vtable_s *vtable);
+
+  /* Get the panel power status (0: full off). */
+
+  int (*getpower)(FAR struct fb_vtable_s *vtable);
+
+  /* Enable/disable panel power (0: full off). */
+
+  int (*setpower)(FAR struct fb_vtable_s *vtable, int power);
```

Comment:
It does not seem correct for the Framebuffer API to handle these operations on the display device.
Why not make these go through the LCD APIs?

Reply:
LCD cannot be operated using framebuffer mode only. If both are enabled, the code is redundant.

Reply:
I believe the expected method for powering off the display device according to the Framebuffer API is through `fb_uninitialize`, but nobody is currently calling it.

https://github.com/apache/incubator-nuttx/blob/60b2a0e2a03878198db24a451bfe115a7d5e0e54/include/nuttx/video/fb.h#L638-L653

What about renaming the `FBIO[GET/SET]_POWER` IOCTL commands for ones that something like `FBIO_[INIT/DEINIT]`? Then the implementation for these commands would rely on the `up_fbinitialize`/`up_fbuninitialize` APIs. Does this work out right?

Reply:
FBIO[GET/SET]_POWER was used to control panel backlight.

As for the framerate configuration,  it can be used for AOD dispalys.

Reply:
> It does not seem correct for the Framebuffer API to handle these operations on the display device.
> Why not make these go through the LCD APIs?

Here is several reasons:

1. FB and LCD driver interface is already separated in the current design(e.g. FBIOPUT_CURSOR v.s. LCDDEVIO_SETCURSOR). So, the new IOCTL(e.g. FBIOSET_POWER) reduce the difference between two interfaces.
2. If we expose the display related operation into two device nodes, it make the userspace GUI library integration harder, because:
   - The library need to manage two handles instead one
   - It's hard to identify fb/lcd pair if the device support multiple display

> I believe the expected method for powering off the display device according to the Framebuffer API is through `fb_uninitialize`, but nobody is currently calling it.

Two problems here:
1. LCD driver already expose LCDDEVIO_GETPOWER/LCDDEVIO_SETPOWER
2. FBIOSET_POWER accept an integer argument to specify the power level, but is strange to pass the power level to FBIO_INIT
2. More important, up_fbuninitialize will destroy device node, and then is impossible to reenable FB again